phpunit-integration-multisite.xml.dist

```diff
@@ -23,6 +23,12 @@
 		</testsuite>
 	</testsuites>
 
+	<groups>
+  		<exclude>
+  			<group>ms-excluded</group>
+  		</exclude>
+  	</groups>
+
```

Comment:
This is a mix of tabs and spaces. Should only use tabs.

Reply:
Fixed